internal/querycoordv2/handlers.go

```diff
@@ -332,7 +332,7 @@ func (s *Server) fillReplicaInfo(replica *meta.Replica, withShardNodes bool) (*m
 			leaderInfo = s.nodeMgr.Get(leader)
 		}
 		if leaderInfo == nil {
-			msg := fmt.Sprintf("failed to get shard leader for shard %s, the collection not loaded or leader is offline", channel)
```

Comment:
why remove this

Reply:
the removed part is noisy, in fact this could be node offline only, and the merr type will indicate this

Reply:
the removed part is noisy, in fact this could be node offline only, and the merr type has indicated what the error is